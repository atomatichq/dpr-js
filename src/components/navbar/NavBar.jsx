import React from 'react'
import { Navbar } from 'react-bootstrap'
import BrandLogo from './brand-small-logo.png'
import './NavBar.css'

class NavBar extends React.Component {

  constructor(props) {
    super(props)
  }

  render() {
    return (
      <Navbar className="navbar navbar-light bg-faded mb-5" role="navigation">
        <div className="container">
          <Navbar.Header>
            <Navbar.Brand>
              <a href="/">
                <img alt="Brand" src={BrandLogo} />
              </a>
            </Navbar.Brand>
          </Navbar.Header>
        </div>
      </Navbar>
    )
  }
}

export default NavBar
