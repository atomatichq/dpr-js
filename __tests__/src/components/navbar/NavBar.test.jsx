import React from 'react'
import renderer from 'react-test-renderer'
import NavBar from '../../../../src/components/navbar/NavBar'

test('Rendered output matches previously created snapshot', () => {
  const component = renderer.create(<NavBar />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
