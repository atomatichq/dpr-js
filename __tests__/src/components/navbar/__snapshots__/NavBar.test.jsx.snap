exports[`test Rendered output matches previously created snapshot 1`] = `
<nav
  className="navbar navbar-light bg-faded mb-5 navbar navbar-default"
  role="navigation">
  <div
    className="container">
    <div
      className="container">
      <div
        className="navbar-header">
        <a
          className="navbar-brand"
          href="/">
          <img
            alt="Brand"
            src="test-file-stub" />
        </a>
      </div>
    </div>
  </div>
</nav>
`;
